batchconsumer/writer.go

```diff
@@ -30,11 +31,12 @@ type batchedWriter struct {
 	lastProcessedSeq kcl.SequencePair
 }
 
-func NewBatchedWriter(config Config, sender Sender, log kv.KayveeLogger) *batchedWriter {
+func NewBatchedWriter(config Config, sender Sender, log kv.KayveeLogger, failedLogsFile kv.KayveeLogger) *batchedWriter {
```

Comment:
Hmm, does the first logger have to be passed it?  It feels a little strange asking the consumers of this lib to pass in two loggers.  Perhaps the first stderr-logger can be created internally?

Reply:
945ed31